samcli/commands/pipeline/bootstrap/guided_context.py

```diff
@@ -33,81 +36,195 @@ def __init__(
         self.pipeline_ip_range = pipeline_ip_range
         self.region = region
 
+    def _prompt_stage_name(self) -> None:
+        click.echo(
+            "Enter a name for the stage you want to bootstrap. This will be referenced later "
+            "when generating a Pipeline Config File with Pipeline Init."
```

Comment:
We should only capitalize proper nouns. Are we using "config" for prompts versus "configuration"? Not sure whether we want to add backticks to CLI commands.

```suggestion
            "when generating a pipeline configuration file with pipeline init."
```